### PR TITLE
Feature/setup docker for development

### PR DESCRIPTION
**Pull request by fredrbus**
## Related Issue
Closes `https://trello.com/c/1gWYT4xl/22-use-volumes-in-docker`

## Proposed changes
- Adds a Docker image and compose file for development

## Additional info
- Hot reloading works

## How has this been tested?
- [x] User testing

## Checklist
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Jorgro** [comment]
I get this error when trying to test it:
Screenshot 2021-09-20 at 15 11 50

**Review by Jorgro — APPROVED** [review]
Worked for me :D

File: README.md [hunk 1]
```diff
@@ -52,3 +52,17 @@ If you only want to run the GraphDB-database, replace `docker-compose-backend.ym
...
+
+To use the development container, follow these steps:
+
+1. Make sure the API is available on http://localhost:3001 (follow the steps above)
+
+2. Copy the `frontend/.env.example`-file, and rename the copy `.env` (if you haven't done so already).
+
+3. In `frontend/`, run `docker-compose -f docker-compose-dev.yml up --build`
+
+4. The app should now be accesible at https://localhost:3000S
```
Comment (espandre) [inline README.md:68]: `https://localhost:3000S` -- fatfingering?

**Review by espandre — APPROVED** [review]
LGTM through cursory testing of the frontend.

**Pull request merged**